sql/20190505185912_change_text_to_timestamp_clients_table.sql

```diff
@@ -0,0 +1,3 @@
+-- migrate:up
+ALTER TABLE clients ALTER COLUMN "birth_date" TYPE DATE
+    USING to_date("birth_date", 'YYYY-MM-DD');
```

Comment:
@BarracudaPff 'cause here added new migration, clients don't wok with DATE at this version. Clients use TEXT.

Reply:
ok